index.js

```diff
@@ -253,6 +254,14 @@ function init() {
             buildBundles(site.bundles);
             // generates the pages
             buildPages(site.pages);
+            // generated pages using glob.
+            const metaJsons = glob.sync('live-examples/**/meta.json', {});
+            for (const metaJson of metaJsons) {
+                const file = fse.readJsonSync(metaJson);
```

Comment:
For this line and inside the loop, I would rather use `let`

Reply:
Hmm, why `let`? It is not like the variable will be reassigned to other thing.

Reply:
I generally like to reserve the use of `const` to things like config, file paths etc. Peppering them throughout a code base feels like asking for trouble ;)

It feels unnecessary to define those as constants.

Reply:
Hmm, I feel quite the opposite because using `const` shows the intention of not reassigning the variable to something else. I will only use `let` if I'm sure I'm going to reassign this to something else. This is so that if we accidentally reassign something else to this variable, the runtime (Node in this case) will throw an error instead of silently mutating the variable.

What do you think?

Reply:
You convinced me :)